packages/esbuild-bin-loader/package.json

```diff
@@ -10,6 +10,6 @@
     "build": "tsup"
   },
   "dependencies": {
-    "fuels": "0.74.0"
+    "fuels": "0.0.0-pr-2017-20240410091814"
```

Comment:
I do not recommend releasing something pointing to a temporary tag, unless you know what you're doing and plan to update it again soon.


We closed the `0.52` PR and should soon delete the tag; which can break this.

Further development is taking place at:
 - https://github.com/FuelLabs/fuels-ts/pull/2045

Reply:
Alright, then we wait for the release of the ts sdk

Reply:
So that you know, the work on `forc` @ `0.53.0` is practically finished:
 - https://github.com/FuelLabs/fuels-ts/pull/2045

There is a ([automagically](https://github.com/FuelLabs/fuels-ts/pull/2045#issuecomment-2050147531)) PR open on the Wallet repo to update to it:
 - https://github.com/FuelLabs/fuels-wallet/pull/1229

We should consider setting up this automatic routine for the bridge as well. 🙂 

---

For now, you could update things here to use [the same tag as the Wallet](https://github.com/FuelLabs/fuels-wallet/pull/1229/files#diff-f28ff95bfbcd3ae5a7bf07043c5ff1d3f9e2ecf7275bc30df652534bcc603e80R13):

```
"fuels": "0.0.0-rc-2045-20240411194225"
```

> [!NOTE]
> One thing to know is that the UNIX suffix on this tag changes on each new commit to the former TS SDK PR.